### PR TITLE
fix(curriculum): video challenge navigation

### LINKED ISSUE #39124: a11y: keyboard navigation on video challenges
**Describe the bug**
There is no visual indication that you have reached the radio buttons or the 'Check your answer' button via tab navigation.

**To Reproduce**
1. Go to https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/introduction-why-program
2. Tab through the page
3. See the embedded video and links get outlined
4. Once you've reached 'Install Python on MacOS' press tab again
5. See that nothing is outlined

**Expected behavior**
The radio button group should be outlined.  The current selection is indicted by the dot, but not that you have selected the buttons.  The 'Check your answer' button also should get outlined when you reach it.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39124

### CODE REVIEW

**Comment by Sky020** [comment]
I will squash the mess, once the checks all pass...

**Comment by ojeytonwilliams** [comment]
I'm mostly thinking out loud, but perhaps something like [this](https://w3.org/WAI/tutorials/forms/grouping/#radio-buttons).  So, rather than make the button group something you can tab onto, instead you outline the first button and then use arrow keys to move between the radio buttons.

I do really like the style in this PR, though. It makes it easy to see which button is selected and looks great.

**Comment by Sky020** [comment]
Unfortunately, because the `input` element is hidden, and the span is the new _'input'_, once the radio is highlighted, tab must be selected again, before the arrows can be used for navigation.

File: client/src/templates/Challenges/video/Show.js [hunk 1]
```diff
@@ -222,35 +222,35 @@ export class Project extends Component {
                 <PrismFormatted text={text} />
                 <Spacer />
                 <ObserveKeys>
-                  <fieldset className='video-quiz-options'>
+                  <div className='video-quiz-options'>
```
Comment (ojeytonwilliams) [inline client/src/templates/Challenges/video/Show.js]: I was just experimenting here, `fieldset` was probably better.

File: client/src/templates/Challenges/video/Show.js [hunk 2]
```diff
@@ -222,35 +222,35 @@ export class Project extends Component {
...
                     {answers.map((option, index) => (
                       // answers are static and have no natural id property, so
                       // index should be fine as a key:
-                      <label className='video-quiz-option-label' key={index}>
-                        <span
+                      <label
+                        className={`video-quiz-option-label ${
+                          this.state.selectedOption === index
+                            ? 'outlined'
```
Comment (ojeytonwilliams) [inline client/src/templates/Challenges/video/Show.js:232]: There's no css that hooks onto this yet (and I didn't name it well), but this approach seemed like an easy way to highlight the whole box without adding a tabindex.

Comment (Sky020) [reply client/src/templates/Challenges/video/Show.js]: This was a good way to help visualise the selection.

**Comment by Sky020** [comment]
@ojeytonwilliams Oliver, this is ready. Forgive the mess - I will clean up, if you think it is ready.

**Pull request closed without merging**